UnityProject/Assets/Scripts/Items/ItemAttributes.cs

```diff
@@ -415,72 +423,64 @@ private static ItemSize getItemSize(string s)
 		}
 	}
 
-	//Below methods add a code to the start of the sprite reference to indicate which spritesheet to use:
-	//1 = items
-	//2 = clothing
-	//3 = guns
+	/// <summary>
+	/// Adds a code to the start of the sprite reference to indicate which spritesheet to use
+	/// </summary>
+	/// <returns>The reference with the code pre-concatenated</returns>
 	public int NetworkInHandRefLeft()
 	{
 		if (inHandReferenceLeft == -1)
 		{
 			return -1;
 		}
 
-		string code = SpriteTypeCode();
-		string newRef = code + inHandReferenceLeft;
-		int i = -1;
-		int.TryParse(newRef, out i);
-		return i;
+		string newRef = SpriteTypeCode() + inHandReferenceLeft;
+		return int.TryParse(newRef, out int i) ? i : -1;
 	}
 
+	/// <summary>
+	/// Adds a code to the start of the sprite reference to indicate which spritesheet to use
+	/// </summary>
+	/// <returns>The reference with the code pre-concatenated</returns>
 	public int NetworkInHandRefRight()
 	{
 		if (inHandReferenceRight == -1)
 		{
 			return -1;
 		}
 
-		string code = SpriteTypeCode();
-		string newRef = code + inHandReferenceRight;
-		int i = -1;
-		int.TryParse(newRef, out i);
-		return i;
+		string newRef = SpriteTypeCode() + inHandReferenceRight;
+		return int.TryParse(newRef, out int i) ? i : -1;
 	}
 
+
 	private string SpriteTypeCode()
 	{
 		int i = -1;
 		switch (spriteType)
 		{
-			case SpriteType.Items:
-				i = 1;
-				break;
-			case SpriteType.Clothing:
-				i = 2;
-				break;
-			case SpriteType.Guns:
-				i = 3;
-				break;
+			case SpriteType.Items   : i = 1; break;
+			case SpriteType.Clothing: i = 2; break;
+			case SpriteType.Guns    : i = 3; break;
 		}
 		return i.ToString();
 	}
 
 	public void OnHoverStart()
 	{
 		// Show the parenthesis for an item's description only if the item has a description
-		if (itemDescription.Length > 0)
-		{
-			UIManager.SetToolTip = itemName + " (" + itemDescription + ")";
-		}
-		else
-		{
-			UIManager.SetToolTip = itemName;
+		string tooltip = itemName;
+
+		if (!String.IsNullOrEmpty(itemDescription)) {
+			itemName += $" ({itemDescription})";
```

Comment:
Don't append item's description to its name:
![image](https://user-images.githubusercontent.com/10403536/56909239-65152000-6a97-11e9-8f84-42cfc30e1fee.png)